### PR TITLE
Support wait for prune in the applier

### PR DESCRIPTION
**Pull request by mortent**

### CODE REVIEW

File: pkg/apply/applier.go [hunk 1]
```diff
@@ -159,13 +170,81 @@ func (a *Applier) readAndPrepareObjects() ([]*resource.Info, error) {
 		return nil, err
 	}
 
+	// Fetch the grouping label used for the current set of resources.
+	groupingLabel, err := prune.RetrieveInventoryLabel(inventoryObject.Object)
```
Comment (seans3) [inline pkg/apply/applier.go:174]: Should this be `inventoryLabel` instead of `groupingLabel` ?

File: pkg/apply/applier.go [hunk 2]
```diff
@@ -159,13 +170,81 @@ func (a *Applier) readAndPrepareObjects() ([]*resource.Info, error) {
 		return nil, err
 	}
 
+	// Fetch the grouping label used for the current set of resources.
```
Comment (seans3) [inline pkg/apply/applier.go:173]: Same

**Comment by seans3** [comment]
Looks good. A simple naming (`grouping` to `inventory`) issue.

File: pkg/apply/applier.go (hunk 1 above)
Comment (mortent) [reply pkg/apply/applier.go]: fixed

File: pkg/apply/applier.go (hunk 2 above)
Comment (mortent) [reply pkg/apply/applier.go]: Fixed.

**Pull request merged**